docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
I agree, but this should be something we deal with in a follow-up though; if this works as it is, I'd leave it in the interest of keeping this (mammoth!) patch as simple as we can.